core/vtk/ttkTrackingFromOverlap/ttkTrackingFromOverlap.h

```diff
@@ -149,10 +130,10 @@ class TTKTRACKINGFROMOVERLAP_EXPORT ttkTrackingFromOverlap
   vector<vector<Edges>> levelTimeEdgesTMap; // E_T
   vector<vector<Edges>> timeLevelEdgesNMap; // E_N
 
-  bool needsToAbort() override {
+  bool needsToAbort() /*override*/ {
```

Comment:
in principle this can go away now (but that's ok for this PR, since it's meant to be temporary).